examples/power_spectra_example.py

```diff
@@ -0,0 +1,44 @@
+"""
+===============================
+Power Spectrum of a TimeSeries
+===============================
+
+An example showing how to estimate the power spectrum of a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+import astropy.units as u
+
+import sunpy.timeseries
+from sunpy.data.sample import RHESSI_TIMESERIES
+
+###############################################################################
+# Let's first load a RHESSI TimeSeries from SunPy's sample data.
+# This data contains 9 columns, which are evenly sampled with a time step of 4 
+# seconds. 
+
+ts = sunpy.timeseries.TimeSeries(RHESSI_TIMESERIES)
+
+###############################################################################
+# We now use SciPy's periodogram(~scipy.signal.lombscargle) to estimate the power 
```

Comment:
Wait... `signal.periodogram` is the same than `signal.lombscargle`? If so, then you could have used the previous series that was not evenly spaced.

Reply:
@dpshelio Its not the same , I'll just change it in the next commit .
Sorry @dpshelio , I am not that acquainted with different types of periodogram. Should I use a normal `signal.periodogram` or a `signal.lombscargle`?

Reply:
Sorry @dpshelio , I am not that well acquainted with lomb scargle periodogram . Is there any advantage of using `signal.lombscargle` over `signal.periodogram`?

Reply:
They are not the same, that should point to `scipy.signal.periodogram` instead.